docs/contributing-guide/1. getting-set-up.md

```diff
@@ -12,9 +12,9 @@
 
 ### Prerequisites
 
-- Ensure node is installed. [Node.js (16.19.0 and above)](https://nodejs.org/en/download/).
-- (Recommended) install vscode. [Visual Studio Code (Stable Build)](https://code.visualstudio.com/Download).
-- Install [rush](https://rushjs.io/), in a terminal of your choosing, run:
+- Install [Node.js (16.19.0 and above)](https://nodejs.org/en/download/).
```

Comment:
We should probably leave this as is; they may already have node 16 installed.

Reply:
I think it should be fine since user have already installed VS Code, Rush.. etc